weaknesses/MASVS-STORAGE/MASWE-0008.md

```diff
@@ -1,29 +1,28 @@
 ---
-title: Device Access Security Policy Not Enforced
+title: Secured Device Detection Not Implemented
```

Comment:
or "Missing Device Secure Lock Verification Implementation". Any other ideas?

Reply:
I put "Missing Device Secure Lock Verification Implementation" for now